tests/infer/mcmc/test_nuts.py

```diff
@@ -80,13 +90,23 @@ def model(data):
         y = pyro.sample('y', dist.Bernoulli(logits=(coefs * data).sum(-1)), obs=labels)
         return y
 
-    nuts_kernel = NUTS(model, step_size=0.0855)
+    nuts_kernel = NUTS(model, step_size, adapt_step_size, adapt_mass_matrix, full_mass)
     mcmc_run = MCMC(nuts_kernel, num_samples=500, warmup_steps=100).run(data)
     posterior = EmpiricalMarginal(mcmc_run, sites='beta')
     assert_equal(rmse(true_coefs, posterior.mean).item(), 0.0, prec=0.1)
 
 
-def test_beta_bernoulli():
+@pytest.mark.parametrize(
```

Comment:
Our HMC tests are consuming more test time than earlier, see below for some of our most expensive unit tests. Should we just use the default parameters (i.e. adaptation = True) on most of these tests, and only use the full parameterized test on one example each in HMC and NUTS? I think that should give us a good balance between coverage and test times, at least until we make HMC faster. :) 

```python
136.68s call     tests/infer/mcmc/test_nuts.py::test_beta_bernoulli[0.02-False-True-False]
107.39s call     tests/infer/mcmc/test_nuts.py::test_gamma_normal[0.01-False-True-False]
51.55s call     tests/infer/mcmc/test_nuts.py::test_nuts_conjugate_gaussian[dim=10_chain-len=4_num_obs=1]
40.83s call     tests/infer/mcmc/test_nuts.py::test_logistic_regression[0.0855-False-True-False]
39.48s call     tests/infer/mcmc/test_hmc.py::test_beta_bernoulli[None-1-None-True-False-False]
29.94s call     tests/infer/test_abstract_infer.py::test_posterior_predictive
29.25s call     tests/infer/mcmc/test_hmc.py::test_gamma_normal[None-1-None-True-False-False]
24.20s call     tests/infer/mcmc/test_hmc.py::test_logistic_regression[None-1-None-True-False-False]
23.35s call     tests/infer/mcmc/test_hmc.py::test_bernoulli_latent_model[True]
23.29s call     tests/infer/mcmc/test_nuts.py::test_beta_bernoulli[0.02-False-False-False]
22.85s call     tests/infer/mcmc/test_nuts.py::test_bernoulli_latent_model
21.22s call     tests/infer/mcmc/test_nuts.py::test_nuts_conjugate_gaussian[dim=10_chain-len=3_num_obs=1]
20.41s call     tests/infer/mcmc/test_nuts.py::test_gamma_beta
19.91s call     tests/infer/mcmc/test_hmc.py::test_hmc_conjugate_gaussian[dim=10_chain-len=4_num_obs=1]
19.60s call     tests/infer/mcmc/test_hmc.py::test_dirichlet_categorical[None-1-None-True-False-False]
```